packages/create-next-app/templates/app-tw/ts/.prettierrc.json

```
@@ -0,0 +1,3 @@
+{
+  "singleQuote": false
+}
```

```suggestion
// Feel free to delete or ignore this file if you're not using `prettier`.
{
  "singleQuote": false
}
```
I wanted to add a comment like this, but `.json` files don't allow comments.